upgrade_tests/helpers/scenarios_utils.py

```diff
@@ -0,0 +1,181 @@
+"""All the helper functions, needed for scenarios test case automation to be
+added here"""
+import os
+import pickle
+import sys
+import time
+
+from automation_tools import manage_daemon
+from automation_tools.bz import bz_bug_is_open
+from automation_tools.satellite6.hammer import (
+    get_attribute_value,
+    hammer,
+)
+from upgrade.helpers.docker import remove_all_docker_containers
+from upgrade.helpers.rhevm import (
+    create_rhevm_instance,
+    get_rhevm_client,
+    wait_till_rhevm_instance_status
+)
+from automation_tools.satellite6.upgrade.tools import logger
```

Comment:
You have logger in upgrades itself. Import from there, logger from automation tools will be deleted soon.